README.md

```diff
@@ -688,6 +688,41 @@ Example ESLint configuration:
 }
 ```
 
+### [unicorn/template-indent]
+
+**This rule can be used with certain options.**
+
+This rule will automatically fix the indentation of multiline string templates, to keep them in alignment with the code they are found in. A configurable whitelist is used to ensure no whitespace-sensitive strings are edited.
+
+The conflict is in templates tagged with an `html` function, in which case both Prettier and Unicorn try to indent the template.
```

Comment:
Prettier deals with:

- html
- graphql
- css
- markdown

Using various tags, functions and comments. See: https://github.com/prettier/prettier/tree/main/src/language-js/embed

Reply:
How about now?